### ISSUE TITLE
Switch without a timer

### ISSUE THREAD

**Issue by Marfre888** [nitaybz/homebridge-delay-switch#26]
Hi,

Firstly I was to say that I love this plugin. It's so simple and yet has been essential to making my timers work when more than one device triggers a light or accessory.

I'm looking for a state switch (ie: I want to save the last state of a device so accessory can return to that state after event has passed, in this case On = device is on and Off = device is off). I have it all figured out, but I'm struggling to find a switch plugin that isn't on a timer.

TLDR; I want to add a switch that doesn't expire with a timer.

Is there any way of going this with this plugin?

**Comment by nitaybz** [comment]
[homebridge-dummy](https://www.npmjs.com/package/homebridge-dummy) is your go to plugin for that.

hope it helps
